M2/Macaulay2/packages/SimpleDoc.m2

```diff
@@ -32,7 +32,7 @@ doc String := str -> (
     document \ (
 	if all(parsed, elt -> instance(elt, Node)) then apply(parsed, node -> toList node)
 	else if not any(parsed, elt -> instance(elt, Node)) then {parsed}
-	else error("expected either a single or a list of documentation nodes")))
+	else error("expected either a single or a list of documentation nodes"));)
```

Comment:
"a single" what?  I think this construction doesn't work.  Try "a documentation node or a list of documentation nodes".

Reply:
how about "one or more documentation nodes"?

Reply:
Too ambiguous: it could mean a list of length greater than 0 is the only thing accepted.  Or it could mean a sequence of nodes is accepted.  Or a set of nodes.